src/Controller/Admin/Asset/AssetController.php

```diff
@@ -1241,6 +1241,13 @@ public function downloadImageThumbnailAction(Request $request): BinaryFileRespon
         }
 
         if ($thumbnail) {
+            $thumbnailConfig = $thumbnail->getConfig();
+            if ($thumbnailConfig->getFormat() === 'SOURCE' &&
+                $autoFromatConfigs = $thumbnailConfig->getAutoFormatThumbnailConfigs()) {
+                $autoFromatConfig = current($autoFromatConfigs);
```

Comment:
a typo in the var name

```suggestion
                $autoFormatConfigs = $thumbnailConfig->getAutoFormatThumbnailConfigs()) {
                $autoFormatConfig = current($autoFormatConfigs);
```